### PR TITLE
docs - updated workload_mgmt_resgroups to support cgroup v1 on rhel9 (6X)

### PR DESCRIPTION
**Pull request by Tao-T**
libcgroup/libcgroup-utils packages are official removed since RHEL9. A new service is provided in the document to support gpdb6 with cgroup v1 on RHEL 9.x

edit: I have added a staging link: https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/mireia-rg-6/greenplum-database/admin_guide-workload_mgmt_resgroups.html#configuring-and-using-resource-groups

### CODE REVIEW

**Review by mperezfuster — APPROVED** [review]
I have submitted a couple of edits. Thanks!

File: gpdb-doc/markdown/admin_guide/workload_mgmt_resgroups.html.md [hunk 1]
```diff
@@ -359,27 +334,46 @@ Complete the following tasks on each node in your Greenplum Database cluster to
...
+        ```
+        cgconfigparser -l /etc/cgconfig.conf 
+        ```
+    -   Redhat/CentOS 6.x systems:
+
+        ```
+        service cgconfig start 
+        ```
+
+1.  To automatically recreate Greenplum Database required cgroup hierarchies and parameters when your system is restarted, configure your system to enable the Linux cgroup service daemon `cgconfig.service` \(Redhat/CentOS 7.x/8.x\) or `cgconfig` \(Redhat/CentOS 6.x\) at node start-up. To ensure the configuration is persisten after reboot, run the following commands as user root:
```
Comment (dyozie) [inline gpdb-doc/markdown/admin_guide/workload_mgmt_resgroups.html.md:349]: persisten -> persistent

File: gpdb-doc/markdown/admin_guide/workload_mgmt_resgroups.html.md [hunk 2]
```diff
@@ -390,26 +384,41 @@ Complete the following tasks on each node in your Greenplum Database cluster to
...
+#### Redhat 9.x
+
+If you are using Redhat 9.x, the `libcgroup` and `libcgroup-tools` packages are not available with the operating system. In this scenario, you must manually create a service that automatically recreates the cgroup hierarchies after a system boot. Add the following bash script for systemd so it runs automatically during system startup. Perform the following steps as user root:
+
+1. Create `greenplum-cgroup-v1-config.service`
+   ```
+   vim /etc/systemd/system/greenplum-cgroup-v1-config.service
+   ```
+
+2. Write the following content into `greenplum-cgroup-v1-config.service`, if the user is not `gpadmin`, replace it with the appropriate user.
```
Comment (dyozie) [inline gpdb-doc/markdown/admin_guide/workload_mgmt_resgroups.html.md:396]: , if -> . If

**Review by dyozie — CHANGES_REQUESTED** [review]

**Comment by Tao-T** [comment]
For the commands mentioned in the document, `sudo` is not unified. Some are still prefixed with `sudo` while the others have `sudo` removed.

**Comment by haolinw** [comment]
I have tried Redhat 9.x script on rocky9, it works fine.

**Pull request merged**